src/pages/public_registration/PublicGeneralContent.tsx

```diff
@@ -241,7 +270,20 @@ export const PublicGeneralContent = ({ registration }: PublicRegistrationContent
               <PublicSeries publicationContext={publicationContext as ReportPublicationContext} />
             </>
           ) : isChapter(publicationInstance.type) ? (
```

Comment:
skal dette bare vises for `Chapter*` typene? Vi har samme visning for noen andre typer også i dag, feks korrigendum: https://dev.nva.sikt.no/registration/018c8170241d-58986526-5836-40e3-b2d2-47599e4a76ed
I det tilfellet er det tidsskrift i stedet for utgiver da 🤔

Reply:
Skal høre med PO

Reply:
Trenger ikke å vise publisher info for korrigendum